examples/moduleExample/app/src/main/java/io/realm/examples/appmodules/model/Pig.java

```diff
@@ -16,18 +16,12 @@
 
 package io.realm.examples.appmodules.model;
 
-import io.realm.RealmList;
 import io.realm.RealmObject;
-import io.realm.examples.librarymodules.model.Dog;
 
 public class Pig extends RealmObject {
 
     private String name;
 
-    // It is possible for model classes to to reference library model classes as long
-    // as they all are included in the schema when opening the Realm.
-    private RealmList<Dog> afraidOf = new RealmList<>();
```

Comment:
It is no longer possible to use library model classes in classes of the default module?

Reply:
It is, but it is just impossible to show in this example at the same time as showing how the default config works.

Reply:
So instead of having an example that is only showing a pretty small cornercase, I thought it was better to show how the default settings work.

Reply:
You can, but you should include the library module as well as the default. since the default scope doesn’t include implicitly the library models